plutus-extra/src/Plutus/Contract/Test/Extra.hs

```diff
@@ -469,6 +471,30 @@ utxoAtComputedAddress contract inst addressGetter cont =
               utxoMap = view (AM.fundsAt addr) am
           cont w addr utxoMap
 
+{- | Extract UTxOs at an address and perform a validation computation on it.
+
+  @since 4.2
+-}
+utxoAtAddress ::
+  forall
+    (effs :: [Type -> Type])
+    (w :: Type)
+    (s :: Row Type)
+    (e :: Type)
+    (a :: Type)
+    (contract :: Type -> Row Type -> Type -> Type -> Type).
+  ( effs ~ '[Reader InitialDistribution, Error EmulatorFoldErr, Writer (Doc Void)]
```

Comment:
Could this be inlined into the signature? So instead of 

```haskell
(UtxoMap -> Eff effs Bool)
```

we would have

```haskell
(UtxoMap -> Eff '[Reader InitialDistribution, Error EmulatorFoldErr, Writer (Doc Void)] Bool)
```

It's longer, but the rigidity constraint there seems an odd choice.

Reply:
It's just a stylistic choice. I've changed it to what you suggested.